apache/superset
# fix(database): make to display validation error msg when all cases

### SUMMARY
The dynamic form to connect to PostgreSQL is not returning connection errors to the user.
No errors display when the error_type is `GENERIC_DB_ENGINE_ERROR ` and so error alerts are displayed at this case.

### BEFORE/AFTER SCREENSHOTS OR ANIMATED GIF
BEFORE:

<removed by @eschutho>

AFTER:

https://user-images.githubusercontent.com/47900232/169852227-e3c1f0ab-d5c5-491c-a1c2-57d43f70bfdb.mov

### TESTING INSTRUCTIONS
**How to reproduce the bug**

Please follow the steps with this sample data.
```

```

1. Hover over the + icon > Data > Connect database.
2. Select PostgreSQL.
3. Input everything correctly with the exception of the DB name.
4. Click on CONNECT. No error is shown.
5. Fix the DB name, and now enter an incorrect username.
6. Click on CONNECT. No error is shown.
7. Replace the password to an incorrect value.
8. Click on CONNECT. No error is shown.

### ADDITIONAL INFORMATION
